velox/connectors/hive/storage_adapters/abfs/tests/AbfsFileSystemTest.cpp

```diff
@@ -47,6 +50,8 @@ class AbfsFileSystemTest : public testing::Test {
   }
 
   void SetUp() override {
+    AbfsConfig::setUpTestWriteClient(
```

Comment:
This is error-prone since we are setting a single static variable and different `SetUp` calls per test can initialize different instances concurrently.
We should call this with `static void SetUpTestSuite()` and the tearDown below with `static void TearDownTestSuite()`

Reply:
Thanks, updated.